website/docs/cli-guide/master.adoc

```diff
@@ -0,0 +1,83 @@
+include::topics/templates/document-attributes.adoc[]
+[id="cli-guide"]
+= CLI Guide
+
+:toc:
+:toclevels: 4
+:numbered:
+:imagesdir: topics/images
+:context: cli-guide
+:cli-guide:
+
+//Inclusive language statement
+include::topics/making-open-source-more-inclusive.adoc[]
+
+== Introduction
+
+// About the {UserCLIBookName}
+include::topics/about-cli-guide.adoc[leveloffset=+2]
+
+// About {ProductName}
+include::topics/what-is-the-toolkit.adoc[leveloffset=+2]
+
+// About the {CLINameTitle}
+include::topics/about-cli.adoc[leveloffset=+2]
+
+== Installing and Running the CLI
+
+// Install the CLI
+include::topics/installing-web-console-or-cli-tool.adoc[leveloffset=+2]
+
+// Run the CLI
+include::topics/proc_cli-execute.adoc[leveloffset=+2]
```

Comment:
Are you planning to adopt this naming convention for modules? If so, you'll need to spend some time updating the other modules so they are consistent.